src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -17,13 +17,24 @@ public class StrykerOptions
 
         public int AdditionalTimeoutMS { get; }
 
-        public StrykerOptions(string basePath, string reporter, string projectUnderTestNameFilter, int additionalTimeoutMS, string logLevel, bool logToFile)
+        public int ThresholdBreak { get; }
+
+        public int ThresholdLow { get; }
+
+        public int ThresholdHigh { get; }
+
+        public StrykerOptions(string basePath, string reporter, string projectUnderTestNameFilter, int additionalTimeoutMS, string logLevel, bool logToFile, 
+        int thresholdBreak, int thresholdLow, int thresholdHigh)
         {
             BasePath = basePath;
             Reporter = ValidateReporter(reporter);
             ProjectUnderTestNameFilter = projectUnderTestNameFilter;
             AdditionalTimeoutMS = additionalTimeoutMS;
             LogOptions = new LogOptions(ValidateLogLevel(logLevel), logToFile);
+            int[] ThresholdList = ValidateThresholds(new int[3]{thresholdBreak, thresholdLow, thresholdHigh});
```

Comment:
I don't like the idea that you're moving the properties into an array and then retreiving them from the array again. You don't know if the `ValidateThresholds` method hasn't changed the order of the Thresholds. 

It doesn't do anything with the list so we don't need to return it. If we do have to return it, we could consider using a tuple.